msi/GitProduct.wxs

```diff
@@ -290,9 +292,12 @@
       <Custom Action="ExecConfigureSslCAInfo" Before="ExecConfigureCoreAutoCrlf">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
-      <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecPostInstallBat">
+      <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecConfigureFsCache">
```

Comment:
@FrankMeha rather than leaving a bogus comment, why not simply unsubscribe from this thread?